### ISSUE TITLE
Licence?

### ISSUE THREAD

**Issue by Newbytee** [kumajaya/degas-mkbootimg#2]
Hi!

Would it be possible for you to give this a licence?

**Comment by mfn003** [comment]
I'm pretty sure this is under the Apache License... Not sure, though.

**Comment by Newbytee** [comment]
Yeah, I see now that the headers have licencing information. One file lacks it though: https://github.com/kumajaya/degas-mkbootimg/blob/master/unpackbootimg.c
